pkg/restore/restore.go

```diff
@@ -893,6 +916,23 @@ func (ctx *context) restoreItem(obj *unstructured.Unstructured, groupResource sc
 				}
 				obj = updatedObj
 			}
+
+			if shouldRenamePV {
```

Comment:
If you make both suggested changes above, then this could be an `else` to the `if !shouldRenamePV`. I think this would work.